digitalocean/database/resource_database_opensearch_config_test.go

```diff
@@ -0,0 +1,48 @@
+package database_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/digitalocean/terraform-provider-digitalocean/digitalocean/acceptance"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccDigitalOceanDatabaseOpensearchConfig_Basic(t *testing.T) {
+	name := acceptance.RandomTestName()
+	dbConfig := fmt.Sprintf(testAccCheckDigitalOceanDatabaseClusterOpensearch, name, "3.7")
```

Comment:
```suggestion
	dbConfig := fmt.Sprintf(testAccCheckDigitalOceanDatabaseClusterOpensearch, name, "2")
```

3.7 is not a supported Opensearch version:

```
$ doctl databases options versions
Engine        Versions
mysql         [8]
pg            [13,14,15,16]
redis         [7]
kafka         [3.7]
opensearch    [1,2]
mongodb       [5.0,6.0,7.0]
```

Reply:
Good catch! Thanks a lot, Andrew :+1: